Sources/_TestingInternals/Versions.cpp

```diff
@@ -10,11 +10,32 @@
 
 #include "Versions.h"
 
+#if defined(_SWT_TESTING_LIBRARY_VERSION) && !defined(SWT_TESTING_LIBRARY_VERSION)
+#warning _SWT_TESTING_LIBRARY_VERSION is deprecated
+#warning Define SWT_TESTING_LIBRARY_VERSION and optionally SWT_TARGET_TRIPLE instead
```

Comment:
Do we even need to bother deprecating it, since it was never public?

Reply:
This is a note to ourselves so that when Apple's fork of the project upstreams this change, we'll remember to adjust the relevant code.

Reply:
(I'll come back and rip this bit out when the dust has settled.)